Meadowlark-js/backends/meadowlark-mongodb-backend/test/integration/OwnershipSecurity.test.ts

```diff
@@ -87,7 +87,7 @@ describe('given the getById of a document owned by the requestor', () => {
   let result;
 
   const authorizationStrategy = 'OWNERSHIP_BASED';
-  const clientName = 'ThisClient';
+  const clientId = 'ThisClient';
```

Comment:
For integration tests, I kept the old client name for the clientId, for system and http tests, I added client_id to the token and base64 encoded the clientName to get a clientId